pkg/helm/handlers/handler_test.go

```diff
@@ -434,26 +434,37 @@ func TestHelmHandlers_HandleHelmRollbackRelease(t *testing.T) {
 		expectedResponse    string
 		fakeResp            map[string]interface{}
 		expectedContentType string
+		body                string
 		error
 		httpStatusCode int
 	}{
 		{
 			name:                "Invalid chart rollback release test",
 			error:               errors.New("Chart path is invalid"),
+			body:                `{"name": "test", "namespace":"test", "version":1}`,
 			expectedResponse:    `{"error":"Failed to rollback helm releases: Chart path is invalid"}`,
 			httpStatusCode:      http.StatusBadGateway,
 			expectedContentType: "application/json",
 		},
 		{
 			name:             "Valid chart rollback release test",
 			expectedResponse: `{"status":"rollback successful"}`,
+			body:             `{"name": "test", "namespace":"test", "version":1}`,
 			fakeResp: map[string]interface{}{
 				"status": "rollback successful",
 			},
 			expectedContentType: "application/json",
 			error:               nil,
 			httpStatusCode:      http.StatusOK,
 		},
+		{
+			name:                "Invalid body in the request should throw an json parsing error",
+			expectedResponse:    `{"error":"Failed to parse request: json: cannot unmarshal string into Go struct field HelmRequest.version of type int"}`,
+			body:                `{"name": "test", "namespace":"test", "version":"abc"}`,
```

Comment:
what would be the behaviour if `version` field is not present?

Reply:
since it's `int`, default value will be set to `0`

Reply:
`rollback` api should check if version is greater than 0.

Reply:
Done